### PR TITLE
Set namespace in curl command

### PR DESCRIPTION
**Pull request by beckmani**

### CODE REVIEW

**Review by y0sher — CHANGES_REQUESTED** [review]
It seems like we repeat ourselves a lot with this. It'd be better to have a function for running api calls.
it will be more consistent and will reduce errors or untraceable changes in the future. (someone declaring a new test might have problems because of a typo.)
Please refactor the `subprocess.Popen` calls to a separate function and use it instead.

File: tests/test_bs.py [hunk 1]
```diff
@@ -396,7 +396,8 @@ def test_transaction(load_config, setup_clients):
 
     api = 'v1/nonce'
     data = '{"address":"1"}'
-    p = subprocess.Popen(['./kubectl-cmd.sh', '%s' % client_ip, "%s" % data, api], stdin=subprocess.PIPE,
+    namespace = testconfig['namespace']
```
Comment (y0sher) [inline tests/test_bs.py]: why can't we just use `testconfig['namespace']` as the last param?

File: tests/kubectl-cmd.sh [hunk 2]
```diff
@@ -5,4 +5,4 @@ set -xe
 export end_point='http://'$1':9090/'$3
 echo $2
 #kubectl run --quiet curl-isaac --image=tutum/curl -i --tty --restart=Never --rm -n sm -- curl --request POST --data '{ "data": "foo" }' $end_point
```
Comment (y0sher) [inline tests/kubectl-cmd.sh:9]: please remove the comment

File: tests/test_bs.py (hunk 1 above)
Comment (beckmani) [reply tests/test_bs.py]: Just for readability.....

File: tests/kubectl-cmd.sh (hunk 2 above)
Comment (beckmani) [reply tests/kubectl-cmd.sh:9]: This comment should stay. It is an example of what need to be passed

File: tests/test_bs.py (hunk 1 above)
Comment (y0sher) [reply tests/test_bs.py]: IMO there's no big difference in readability

File: tests/kubectl-cmd.sh (hunk 2 above)
Comment (y0sher) [reply tests/kubectl-cmd.sh:9]: 👍  , It'll be nice if we make the comment explanatory instead just throwing it in there.

File: tests/test_bs.py (hunk 1 above)
Comment (beckmani) [reply tests/test_bs.py]: Fixed

File: tests/kubectl-cmd.sh (hunk 2 above)
Comment (beckmani) [reply tests/kubectl-cmd.sh:9]: ok

**Review by y0sher — APPROVED** [review]

**Pull request merged**